src/components/text-editor/__internal__/editor-validation-wrapper/editor-validation-wrapper.style.ts

```diff
@@ -0,0 +1,14 @@
+import styled from "styled-components";
+
+const StyledValidationWrapper = styled.div`
+  margin: 16px 16px 0px 4px;
```

Comment:
**question** (non-blocking): I don't know the answer to this, but since part of the purpose of the PR was to replace hard-coded px values by tokens: should some of these CSS values be tokens too? eg `16px` is `spacing200` and the `14px` font-size is `fontSizes100`.

Reply:
Yep it should have always been 14px as far as I can tell. I incorrectly believed the previous updates done to the CharacterCount was updating this as well as the text alignment so I this could be a `chore` as it's just updating tokens. I'll amend the commit to ensure this is included in any release notes 👍